### PR TITLE
ZEB-872/ fix users table display

### PR DESCRIPTION
**Pull request by MitskevichVS**

### CODE REVIEW

File: client/app/_users/users.html [hunk 1]
```diff
@@ -67,7 +67,12 @@
                                                 <b name="userUsername">{{user.username}}</b>
                                             </td>
                                             <td data-title="Email" name="userEmail" md-cell>{{user.email}}</td>
-                                            <td data-title="First/Last name" name="userFirstLastName" md-cell>{{user.firstName}} {{user.lastName}}</td>
+                                            <td class="users-table-row _name" data-title="First/Last name" name="userFirstLastName" md-cell>
```
Comment (danymill) [inline client/app/_users/users.html]: please use `users-table__row` instead of `users-table-row`

File: client/app/_users/users.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
+.users-table {
+    width: 100%;
+    table-layout: fixed;
+
+    &-row {
+        &._name-wrapper {
+            flex: auto;
+            width: auto;
+            overflow: hidden;
+            text-overflow: ellipsis;
```
Comment (danymill) [inline client/app/_users/users.scss]: Does it work w/o `max-width`? and do you need this here at all?

**Review by danymill — CHANGES_REQUESTED** [review]

File: client/app/_users/users.scss (hunk 2 above)
Comment (MitskevichVS) [reply client/app/_users/users.scss]: fixed

File: client/app/_users/users.html (hunk 1 above)
Comment (MitskevichVS) [reply client/app/_users/users.html]: fixed

**Review by danymill — APPROVED** [review]

**Pull request merged**